### DISCUSSION TITLE
done

### DISCUSSION THREAD

**Discussion by duriann** [community/community#56829] [Copilot Conversations]
### Select Topic Area

Question

### Body

done

**Comment by Akash1134** [comment]
Hi @bolosea ,

People who maintain popular open source projects receive a credit to have 12 months of GitHub Copilot access for free. A maintainer of a popular open source project is defined as someone who has write or admin access to one or more of the most popular open source projects on GitHub. 

Simply visit the GitHub Copilot subscription page to see if you are one of the open source maintainers that meet our criteria for a complimentary subscription. If you do, you should see that you can add GitHub Copilot for no charge. If you see a charge on the purchase page then this means that you do not qualify at this time. Once awarded, if you are still a maintainer of a popular open source project when your initial 12 months subscription expires then you will be able to renew your subscription for free.

For more details, see [Copilot FAQs](https://github.com/pricing#i-work-on-open-source-projects-can-i-get-access-to-github-copilot-for-free:~:text=for%20the%20project%3F-,Copilot,-What%20is%20the)

Thanks!
